.ibm/pipelines/resources/cluster_role_binding/cluster-role-binding-ocm.yaml

```diff
@@ -13,3 +13,9 @@ subjects:
   - kind: ServiceAccount
     name: rhdh-k8s-plugin
     namespace: showcase-ci-nightly
+  - kind: ServiceAccount
+    name: rhdh-k8s-plugin
+    namespace: showcase-operator-nightly
+  - kind: ServiceAccount
+    name: rhdh-k8s-plugin
+    namespace: showcase-op-rbac-nightly
```

Comment:
this config is required for ocm test case and that only runs in showcase not on showcase-rbac, so you may remove `showcase-op-rbac-nightly` namespace.